documentation/css/style.css

```diff
@@ -213,10 +215,13 @@
     height: 30rem;
 }
 
+.li-code{
+    margin: 8px 0;
+}
```

Comment:
I think this tag can be named better, its a suggestion tough no need to change right away

Reply:
Sure.